addons/rating/static/src/js/rating.js

```diff
@@ -0,0 +1,33 @@
+odoo.define ('rating.rating', function (require) {
+'use strict';
+
+var publicWidget = require('web.public.widget');
+
+/**
+ * Rating
+ *
+ * widget for rating, allow to change the rating at time of feedback.
+ */
+publicWidget.registry.rating = publicWidget.Widget.extend({
+    selector: '.o_rating_rating',
+    events: {
+        'click .o_rating': '_onClickSmiley',
+    },
+
+    //--------------------------------------------------------------------------
+    // Handlers
+    //--------------------------------------------------------------------------
+
+    /**
+     * @private
+     * @param {MouseEvent} event
+     */
+    _onClickSmiley: function (ev) {
+        ev.preventDefault();
+        var $target = $(ev.currentTarget);
```

Comment:
isn't there a way to use bootstrap stuff to do this ? 
Have a look at https://getbootstrap.com/docs/4.3/components/buttons/#button-plugin, it might be usefull.

Reply:
@jem-odoo Done.